readthedocs/proxito/tests/test_full.py

```diff
@@ -796,10 +796,42 @@ def test_404_storage_serves_custom_404_sphinx(self, storage_exists, storage_open
         self.assertEqual(response.status_code, 404)
 
     @mock.patch.object(BuildMediaFileSystemStorageTest, 'exists')
-    def test_redirects_to_correct_index(self, storage_exists):
-        """This case is when the project uses a README.html as index."""
+    def test_redirects_to_correct_index_ending_with_slash(self, storage_exists):
+        """When the path ends with a slash, we try README.html as index."""
```

Comment:
Eventually, we should remove this and stop trying `README.html`. That's a pretty legacy decision that I don't think it applies anymore.

We should probably start with new projects and keeping this behavior only for old projects and remove this logic finally after a couple of months.

Reply:
I'm in favor of removing it or documenting it explicitly. This is not expected default functionality from a webserver/host although it isn't a super uncommon reconfiguration.

Reply:
This is a good idea 💡

Reply:
@ericholscher we are already logging this 😉 . Take a look at this https://onenr.io/0LREdxdX5wa

There are more project than I've expected doing this 😞

Reply:
We have 340 projects in .org  and 1 in .com https://onenr.io/0BQ1Gv2BMRx

Reply:
we should continue this at https://github.com/readthedocs/readthedocs.org/issues/9993